index.js

```diff
@@ -154,12 +154,19 @@ exports.middleware = store => next => action => {
   const {type, data} = action
   const {sessions} = store.getState()
   const {activeUid} = sessions
+        // Remove ANSI escape code sequences. Visualization: https://goo.gl/IY8vuU
+  const ANSI_escape_codes = /[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/gm
```

Comment:
These constants won't change, so they shouldn't be defined inside the middleware.